includes/admin/models/class-rop-settings-model.php

```diff
@@ -401,7 +401,7 @@ private function validate_settings( $data ) {
 			}
 
 			$global_settings = new Rop_Global_Settings();
-			$min_hours = 5;
+			$min_hours       = ! empty( get_option( 'rop_data', '' ) ) ? 5 : 12;
```

Comment:
This validation is done every time the user saves the data. I think this trick might work only once (at the first save).

Reply:
@Soare-Robert-Daniel As per the selul comment here - https://github.com/Codeinwp/tweet-old-post/pull/975#issuecomment-2214358933, I've added a new option to identify new users, which seems to work now.